tree/30_generic_methods/file_augeas_commands.cf

```diff
@@ -55,28 +55,28 @@
 # ```
 #
 # @parameter variable_prefix The prefix of the variable name
-# @parameter variable_name   The variable to define, the full name will be variable_prefix.variable_name
```

Comment:
This is most likely not what we want

Reply:
Indeed, this one doesn't work because it does not have a parameter containing a path, shouldn't have been a `file_` method in the first place.

Reply:
We may need to add an exception and move it  into another resource, maybe a dedicated `augeas` resource.